docs/upgrade.md

```diff
@@ -88,6 +88,18 @@ process, for example:
     dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
     ```
 
+# Upgrading to v1.74.0
+
+## Unicode support in user search
+
+This version introduces optional support for an [improved user search dealing with Unicode characters](https://github.com/matrix-org/synapse/pull/14464).
+
+To do so it uses the ICU library through PyICU.
+If you want to take advantage of this feature you need to install PyICU
+but also ICU native dependency and its development headers
+so that PyICU can build since no prebuild wheels is available.
+You can follow [PyICU documentation](https://pypi.org/project/PyICU/) to do so.
```

Comment:
```suggestion
but also the ICU native dependency and its development headers
so that PyICU can build since no prebuilt wheels are available.
You can follow [PyICU documentation](https://pypi.org/project/PyICU/) to do so.
```